driver/examples/configs/baroclinic_c12_orch_cpu.yaml

```diff
@@ -0,0 +1,89 @@
+stencil_config:
+  compilation_config:
+    backend: dace:cpu
+    rebuild: false
+    validate_args: true
+    format_source: false
+    device_sync: false
+initialization:
+  type: baroclinic
+performance_config:
+  performance_mode: true
+  experiment_name: c12_baroclinic
+nx_tile: 12
+nz: 79
+dt_atmos: 225
+minutes: 60
+layout:
+  - 1
+  - 1
+diagnostics_config:
+  path: "output.zarr"
+  names:
+    - u
+    - v
+    - ua
+    - va
+    - pt
+    - delp
+    - qvapor
+    - qliquid
+    - qice
+    - qrain
+    - qsnow
+    - qgraupel
```

Comment:
[nit] since we're not checking the output, we should leave this empty?

Reply:
Right, I just copied this over. Thanks for noticing!